frontend/src/metabase/account/app/components/AccountHeader/AccountHeader.jsx

```diff
@@ -34,11 +35,14 @@ const AccountHeader = ({ user, path, onChangeLocation }) => {
     [hasPasswordChange],
   );
 
+  const userFullName = [user.first_name, user.last_name].join(" ").trim();
```

Comment:
Can't use `user.common_name` since it'll fall back to email when names are null

Reply:
Would it make sense to add a utility to get the full name properly? 🤔 TBH, I'd just go with `user.common_name` and that's it. Not sure if there's a need in `common_name` at all in this case

Reply:
With the current BE change, `common_name` will fall back to email. So it would look something like this.

#### User 1
```yaml
first_name: John
last_name: Wick
email: john.w@hightable.org
common_name: John Wick
```

#### User 2
```yaml
first_name: null
last_name: null
email: jamesb007@mi6.org
common_name: jamesb007@mi6.org
```

and since we're displaying name and email deliberately in this ui. We can't use `common_name`

![image](https://user-images.githubusercontent.com/1937582/173746219-e5228cdb-cac6-4a89-9514-89241bd6f25f.png)

I've been using this `[user.first_name, user.last_name].join(" ").trim()` in a few other places though. Not sure if it's worth being under its own util yet. I'm fine either way.

Reply:
I'm for extracting it into ideally. Ideally something like `getFullName(user: User)`